src/06-mocking-node-api/index.test.ts

```diff
@@ -29,11 +41,33 @@ describe('doStuffByInterval', () => {
   });
 
   test('should set interval with provided callback and timeout', () => {
-    // Write your test here
+    const callback = jest.fn();
+    const interval = 1000;
+
+    doStuffByInterval(callback, interval);
+
+    expect(callback).not.toHaveBeenCalled();
+
+    jest.advanceTimersToNextTimer();
+    expect(callback).toHaveBeenCalledTimes(1);
+
+    jest.advanceTimersToNextTimer();
+    expect(callback).toHaveBeenCalledTimes(2);
```

Comment:
```suggestion
    const timeout = 1000;
    const cb = jest.fn();
    const interval = jest.spyOn(global, 'setInterval');

    doStuffByInterval(cb, timeout);

    expect(interval).toHaveBeenLastCalledWith(cb, timeout);
```

Reply:
Thank you for carefully and thoroughly checking the work and making suggestions!